### PR TITLE
Wiring a first integration test

### PR DESCRIPTION
**Pull request by Fernan2**
The `temping` gem is added to avoid external dependencies
`launchy` added for having `save_and_open_page`
`sudo: false` to reach the new infrastructure of Travis

Also added `rails_helper` and updated the schema with a pending migration...
... and finally did a very basic integration test, to ensure that integration tests work.

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+ENV['RAILS_ENV'] ||= 'test'
+
+require 'spec_helper'
+require 'rspec/rails'
```
Comment (d2bit) [inline spec/rails_helper.rb]: this is also included in `spec_helper`

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+sudo: false
 language: ruby
 cache: bundler
 rvm:
```
Comment (d2bit) [inline .travis.yml]: Maybe it's nice to update all the configuration file (as main projects)
```yml
sudo: false
language: ruby
cache: bundler
rvm:
  - '2.2.2'
before_script:
  - cp test/dummy/config/database.yml.travis test/dummy/config/database.yml
  - bundle exec rake db:setup
addons:
  postgresql: '9.3'
```

File: spec/rails_helper.rb (hunk 1 above)
Comment (Fernan2) [inline spec/rails_helper.rb]: Not anymore... ;-)

**Comment by Fernan2** [comment]
@d2bit done!

**Comment by d2bit** [comment]
:+1:

**Pull request merged**